pkgs/tools/admin/salt/pepper/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib
+, fetchurl
+, python2Packages
+}:
+
+python2Packages.buildPythonApplication rec {
+  name = "salt-pepper-${version}";
+  version = "0.5.0";
+  src = fetchurl {
+    url = "https://github.com/saltstack/pepper/releases/download/${version}/${name}.tar.gz";
+    sha256 = "0gf4v5y1kp16i1na4c9qw7cgrpsh21p8ldv9r6b8gdwcxzadxbck";
+  };
+
+  meta = {
+    description = "A CLI front-end to a running salt-api system";
+    homepage = https://github.com/saltstack/pepper;
+    license = lib.licenses.asl20;
```

Comment:
@Mic92 @FRidh Sorry to bother you both with this again but I am wondering if `salt-pepper` is not a better name for the attribute in all-packages. I am not sure what is the most idiomatic between `pepper`, `salt_pepper` or `salt-pepper`. The executable is named 'pepper' and the python package is named 'salt-pepper'.